hw/dv/data/tests/csr_tests.hjson

```diff
@@ -32,6 +32,11 @@
       run_opts: ["+csr_aliasing"]
       en_run_modes: ["csr_tests_mode"]
     }
+    {
```

Comment:
nit: add newline

Reply:
oh thank you for the reminder, sorry I missed that format.